pkg/scheduler/apis/config/v1beta1/defaults.go

```diff
@@ -44,6 +44,9 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 
 // SetDefaults_KubeSchedulerConfiguration sets additional defaults
 func SetDefaults_KubeSchedulerConfiguration(obj *v1beta1.KubeSchedulerConfiguration) {
+
+	obj.CPUThreads = 16
```

Comment:
do this only if it the pointer is nil

Reply:
should we check ensure CPUThreads > 0 at pkg/scheduler/apis/config/validation.ValidateKubeSchedulerConfiguration?

Reply:
Yes to only if pointer is nil and yes to validation.